projboard/tests/test_article.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+from projboard.models.article import Article, Like, View, Subject, User
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+
+
+class TestArticle:
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_supjects(self):
+        sup = Subject(name='N')
+        sup1 = Subject(name='R')
+        sup.save()
+        sup1.save()
+        return sup, sup1
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_user(self):
+        user = User(name="Anas", nickname="A99", email="Anas@gmail.com", password="Anas123")
+        user1 = User(name="John", nickname="J99", email="John@gmail.com", password="John123")
+        user.save()
+        user1.save()
+        return user, user1
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def create_articles(self, create_user, create_supjects):
+        article = Article(user_id=create_user[0], title=TITLE, content=CONTENT, subject_id=create_supjects[0])
+        article1 = Article(user_id=create_user[1], title="RED HAT", content=CONTENT, subject_id=create_supjects[1])
+        article.save()
+        article1.save()
+        return article, article1
```

Comment:
consider return this  format
articles_list=[]
for i in len(create_user):
    article = Article(user_id=create_user[i], title=TITLE, 
    content=CONTENT, subject_id=create_supjects[i],
    article.save()
    articles_list.append(article)
    
return  articles_list
Do a for loop with indexes of